external/mbedtls/pk_wrap.c

```diff
@@ -112,75 +113,108 @@ static int rsa_verify_wrap(void *ctx, mbedtls_md_type_t md_alg, const unsigned c
 
 static int rsa_sign_wrap(void *ctx, mbedtls_md_type_t md_alg, const unsigned char *hash, size_t hash_len, unsigned char *sig, size_t *sig_len, int (*f_rng)(void *, unsigned char *, size_t), void *p_rng)
 {
-	*sig_len = ((mbedtls_rsa_context *)ctx)->len;
+	int ret;
+	
+#if defined(CONFIG_TLS_WITH_SSS) && defined(CONFIG_SUPPORT_FULL_SECURITY)
+	if (!see_check_keyindex(((mbedtls_rsa_context *) ctx)->key_index)) {
```

Comment:
remove a space between type casting and variable.
Same for every these codes (There are too many)